### PR TITLE
Added: Mergeable configuration

### LINKED ISSUE #102: Create mergeable config
We might want to add a mergeable configuration for PRs so that the PR is created correctly with new entries in the changelog file, a good title, etc. (https://mergeable.readthedocs.io/en/latest/configuration.html)

### PR DESCRIPTION
**Pull request by GoliathLabs**
I added a mergeable configuration to have proper PRs that follow a certain standard (https://mergeable.readthedocs.io/en/latest/configuration.html)

Closes #102

### CODE REVIEW

File: .github/mergeable.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+          enabled: true
+          message: Description matter and should not be empty. Provide detail with **what** was changed, **why** it was changed, and **how** it was changed.
+      - do: dependent
+        changed:
+          file: 'package.json'  # also supports globs expressions
+          required: ['package-lock.json'] # alias: `files` for backward compatibility
+      - do: changeset
+        must_include:
+          regex: 'CHANGELOG.md'
+          message: 'new features or fixes needed to be logged to the changelog'
```
Comment (DasSkelett) [inline .github/mergeable.yml:30]: Do we really want this? We already have a changelog in the release notes.

Comment (GoliathLabs) [reply .github/mergeable.yml]: Removed

File: CHANGELOG.md [hunk 2]
```diff
@@ -18,6 +18,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added proper go linter
 - Fixed DNS proxy upstream lookup loop
 - Updated the documentation to reflect the changes regarding IPv6
+- Added Mergeable config
```
Comment (DasSkelett) [inline CHANGELOG.md]: The above points are no longer  "Latest" but part of the releases 0.5.0-0.5.2. The section needs renaming and this entry should be moved into a new "Latest" section.

**Comment by DasSkelett** [comment]
How will this run? Does this need activating somewhere?

**Comment by GoliathLabs** [comment]
No, should run out of the box

**Review by DasSkelett — APPROVED** [review]
Let's give it a try. We'll see which rules make sense and which are more annoying than helpful and can tune it then.

**Comment by awlx** [comment]
App is approved.

**Pull request merged**